.cbmc-batch/jobs/aws_cryptosdk_sig_abort/aws_cryptosdk_sig_abort_harness.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <aws/cryptosdk/cipher.h>
+#include <cbmc_invariants.h>
+#include <ec_utils.h>
+#include <evp_utils.h>
+#include <make_common_data_structures.h>
+#include <proof_allocators.h>
+#include <proof_helpers/proof_allocators.h>
+
+#include <cipher_openssl.h>
+
+void harness() {
+    /* arguments */
+    struct aws_cryptosdk_sig_ctx *ctx = can_fail_malloc(sizeof(struct aws_cryptosdk_sig_ctx));
+
+    /* assumptions */
+    if (ctx) {
+        ensure_sig_ctx_has_allocated_members(ctx);
+        __CPROVER_assume(ctx->alloc);
+        __CPROVER_assume(aws_cryptosdk_sig_ctx_is_valid_cbmc(ctx));
+    }
+
+    /* saving previous state */
+    EC_KEY *keypair                 = ctx ? ctx->keypair : NULL;
+    int old_keypair_reference_count = ec_key_get_reference_count(keypair);
+    bool keypair_needs_clean_up     = old_keypair_reference_count > 2;
```

Comment:
Why keypair in some of these names and old_keypair in others (since they app appear to be statements about the old key pair)?  Similar for pkey below